samples/net/mqtt_azure/src/test_certs.h

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright (c) 2019 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __TEST_CERTS_H__
+#define __TEST_CERTS_H__
+
+#include <mbedtls/ssl_ciphersuites.h>
+
+static const unsigned char ca_certificate[] =
+/* DigiCert Baltimore Root */
+#if 0
```

Comment:
Still not resolved.

Reply:
I am not following. What did I missed exactly?

Reply:
My point was that there is no point providing two copies of the certificate, in two different formats (especially with at manual `#if 0` switch. I'd stick to one of them, with preference for the PEM format (the one beginning with `-----BEGIN CERTIFICATE-----\r\n`).

The reason for this preference is that I remember from the past, that people were struggling with use of PEM-formated certificates with Zephyr, so it would be good to have a sample showing how to use it. I've tested this one, it just worked.

Reply:
Ok, My intention was there are https://github.com/Azure/azure-iot-sdk-c/blob/master/certs/certs.c three different certs are there. I just want to show which one I am using in this sample.

Reply:
I see, in that case, I'm even more keen to stick to the `DigiCert Baltimore Root`, just as they provide.

Reply:
I would prefer here that we use the PEM file and not embed it to .h file. We have infrastructure to embed a file in source tree into .c file which makes it convenient to use to the end user.